src/components/dialog_preference/index.tsx

```diff
@@ -15,6 +18,7 @@ import { useRecoilValue, useResetRecoilState } from 'recoil'
 
 const PreferenceDialog: FC = () => {
   const isPreferenceDialogOpened = useRecoilValue(isPreferenceDialogOpenedState)
+  const Preference = useRecoilValue(PreferenceState)
```

Comment:
```suggestion
  const preference = useRecoilValue(preferenceState)
```
pが大文字ではなく小文字。 (20行のように)